config_defaults.yml

```diff
@@ -583,8 +583,7 @@ Gemfile:
         - '< 4.0.0'
       - gem: 'puppetlabs_spec_helper'
         version:
-        - '>= 3.0.0'
-        - '< 5.0.0'
+        - '~> 5.0'
```

Comment:
I think the PR title should be adjusted to mention this change as well.

Reply:
Fine with me!